airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -264,45 +215,81 @@ def is_superuser(self):
         """Access all the things"""
         return self.superuser
 
-    def get_similar_users(self, conn):
-        """ Provides all similar user including current user. """
+    def _
 
-        search_base = configuration.conf.get("ldap", "basedn_group")
-        group_filter = configuration.conf.get("ldap", "group_filter")
-        group_name_attr = configuration.conf.get("ldap", "group_name_attr")
+    def _create_custom_super_user_filter():
         superuser_filter = configuration.conf.get("ldap", "superuser_filter")
-        superuser_group = configuration.conf.get("ldap", "superuser_group")
-
         super_user_filter = "(&({0}))".format(superuser_filter)
-        group_user_filter = "(&({0}{1},{2}))".format(group_filter,
-                                                     self.user.username,
-                                                     group_name_attr)
 
-        users = None
+        return super_user_filter
+
+    def _create_group_user_filter(internal_user_group_filter, external_user_group_filter):
```

Comment:
We could remove arguments from this function, and simply call both `_create` functions in its implementation